tests/eval_tests/tools/tools_bad_function_names.py

```
@@ -0,0 +1,463 @@
+def function_13(a: float, b: float) -> float:
+    """
+    :description: Finds the maximum of two numerical inputs.
+    :use_case: Use when the user asks for the "maximum", "larger", or "greater" of two values.
+    :param a: The first number.
+    :param b: The second number.
+    :returns: The larger of the two numbers.
+    """
+    return max(a, b)
+
+@client_tool
+def function_13(a: float, b: float) -> float:
```

Ruff (0.11.9)

Redefinition of unused `function_13` from line 133

(F811)